app/views/admin/settings/forms/_mandatory.html.erb

```diff
@@ -40,18 +40,8 @@
         </div>
       <% end %>
     <% end %>
-
-    <% if current_user.has_role?(:super_admin) %>
-      <div class="crayons-notice p-4 mt-4">
-        <div class="crayons-field">
-          <%= label_tag :confirmation_update, "Confirmation update", class: "crayons-field__label" %>
-          <p class="crayons-field__description">Type the sentence: <strong><%= @confirmation_text %></strong></p>
-          <div class="flex">
-            <%= text_field_tag :confirmation, nil, class: "crayons-textfield flex-1 mr-2", placeholder: "Confirmation text", autocomplete: "off", required: true, pattern: @confirmation_text %>
-            <%= f.submit "Update Settings", class: "crayons-btn align-self-start" %>
-          </div>
-        </div>
-      </div>
-    <% end %>
   </fieldset>
+  <% if current_user.has_role?(:super_admin) %>
```

Comment:
The other partials in the `/admin/customizations/config` view do not have this `<% if current_user.has_role?(:super_admin) %>` conditional in place, but the conditional was present in this partial, so I left it in. @citizen428 advise? I think you last touched this file before I did.

Reply:
As far as I know, all settings inheriting to `Admin::Settings::BaseController`  will go through this authorization check: 

https://github.com/forem/forem/blob/6d2cdeaf41533ff5289e4738783bc7b0e156d60b/app/controllers/admin/settings/base_controller.rb#L22

so I think the if in the partial is redundant, as they won't be able to update anyway. 

I don't think we should hide a submit button, sounds like a dark pattern